packages/react/src/components/Tooltip/Tooltip.js

```diff
@@ -306,6 +311,18 @@ class Tooltip extends Component {
     }
   };
 
+  getAccessibleLabel = (ariaLabel, triggerText, triggerId) => {
+    // if the user provides and aria-label go with that
+    if (ariaLabel) {
+      return { 'aria-label': ariaLabel };
+      // if the user provides triggerText -- set aria-labelledby to that element's ID
+    } else if (triggerText) {
+      return { 'aria-labelledby': triggerId };
+    }
+    // if the user provides neither of those set aria-label to icon-description
+    return { 'aria-label': iconDescription };
```

Comment:
how are you getting `iconDescription` here?

Reply:
Hey, getting back to this now. :D That's just a prop. Are you thinking it should be required if we don't have either `ariaLabel` or `triggerText`?

Reply:
what I mean is it looks like it's undefined. you would need to get the value from `this.props.iconDescription` first